mxd/README.md

```diff
@@ -31,6 +31,10 @@ once and are accessible by all participants.
 For the most bare-bones installation of the dataspace, execute the following commands in a shell:
 
 ```shell
+# We first clone the default branch of this repo
+git clone https://github.com/eclipse-tractusx/tutorial-resources.git
```

Comment:
is this really necessary? first, I think we can expect people, that have any hope of completing the tutorial, to know what Git is, and second, there are other ways to clone a repo (GitHub Desktop, various UI clients, integrated Git client in IntelliJ....).

Personally, i don't think we need this.

Reply:
Ok I agree, the git clone should be clear enough also if it is currently implicit in the tutorial. The suggestion arose after following the tutorial with some colleagues who wanted to understand the functionality of the EDC without being developers themselves.

I actually signed the ECA but committed the code from within GitHub Codespace, which assigns a random E-Mail every session so the Email in the commits doesn't correspond to my ECA Email. 

I think I'll just close this pull request and create a new one with the proposed changes and the right Email.

Thanks!